### PR TITLE
Replace ERD with eGLD

### PR DESCRIPTION
**Pull request by iulianpascalau**
- replaced ERD to eGLD in comments

### CODE REVIEW

**Review by AdoAdoAdo — DISMISSED** [review]

File: systemSmartContractsConfig.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [StakingSystemSCConfig]
-    GenesisNodePrice = "2500000000000000000000" #2.5KERD
-    MinStakeValue = "100000000000000000000" #100ERD
+    GenesisNodePrice = "2500000000000000000000" #2.5K eGLD
+    MinStakeValue = "100000000000000000000" #100 eGLD
```
Comment (sasurobert) [inline systemSmartContractsConfig.toml:3]: Comment is wrong.

**Review by sasurobert — CHANGES_REQUESTED** [review]

File: systemSmartContractsConfig.toml (hunk 1 above)
Comment (iulianpascalau) [reply systemSmartContractsConfig.toml:3]: `100000000000000000000` -> `100,000_000_000_000_000_000 eGLD`

Comment (iulianpascalau) [reply systemSmartContractsConfig.toml]: done, reduced to 0.1 eGLD

**Review by sasurobert — DISMISSED** [review]

File: systemSmartContractsConfig.toml (hunk 1 above)
Comment (AdoAdoAdo) [reply systemSmartContractsConfig.toml]: isn't this too low? I mean the min step is already 100 eGLD, the minStake should at least be one step.

**Review by AdoAdoAdo — CHANGES_REQUESTED** [review]
let's discuss the minStake and decide.

**Review by sasurobert — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

File: systemSmartContractsConfig.toml (hunk 1 above)
Comment (iulianpascalau) [reply systemSmartContractsConfig.toml:3]: reverted

**Pull request merged**